turbinia/config/turbinia_config.py

```diff
@@ -45,28 +82,27 @@
 # NFS or a SAN for Evidence objects.
 SHARED_FILESYSTEM = False
 
-# This will set debugging flags for processes executed by Tasks (for
-# Tasks/binaries that support it).  This could cause performance issues with
-# some tasks, so it is recommended to only set this to True when debugging
-# problems.
-DEBUG_TASKS = False
+###############
+# GCP AND GCE #
+###############
 
-# GCE configuration
 PROJECT = None
 ZONE = None
 INSTANCE = None
 DEVICE_NAME = None
 SCRATCH_PATH = None
-BUCKET_NAME = None
+BUCKET_NAME = 'turbinia'
```

Comment:
Same comment as above WRT to the bucket names.   Also, are both of these needed?  Won't they point to the same thing?

Reply:
Set as None by default.